### ISSUE TITLE
is there 3D support?

### ISSUE THREAD

**Issue by jingpengw** [pytorch/glow#2288]
I see the data format is NCHW, it seems that this will only work with 2D image?
Is there support of 3D, such as NCDHW?

**Comment by jfix71** [comment]
Hi @jingpengw, no, currently there isn't support. 3D convolutions are of interest to us though. Would you be interested in working on this issue?

**Comment by jingpengw** [comment]
yep, would be nice to have it, but this is not my priority right now.

**Comment by jackm321** [comment]
I can take a look at adding support for 3D

**Comment by nickgg** [comment]
@jackm321 what's the status of this?

**Comment by rdzhabarov** [comment]
@jingpengw we added 3d convolution support https://github.com/pytorch/glow/blob/master/tools/ClassGen/NodeGen.cpp#L90
https://github.com/pytorch/glow/blob/master/lib/Backends/Interpreter/InterpreterNodes.cpp#L534

Can you flesh out more details about what you are looking for?
